src/libs/antares/mersenne-twister/include/antares/mersenne-twister/mersenne-twister.hxx

```diff
@@ -38,6 +38,10 @@ inline void MersenneTwister::reset()
     reset(defaultSeed);
 }
 
+inline typename MersenneTwister::Value MersenneTwister::operator()()
+{
+    return this->next();
+}
```

Comment:
Class `MersenneTwister` is not templated, so
1. Place this in a .cpp file
2. Remove keyword `typename`